src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="search-form"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        class="search-form__input"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        id="formInputBig"
+        name="formInputBig"
+      >
+    </form>
+
+    <form
+      action="#"
+      method="post"
+      class="search-form"
+      data-qa="small"
+    >
+      <input
+        type="text"
+        class="search-form__input search-form__input-small"
```

Comment:
Modifiers should have 2 dashes

![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/84296041/dd233b79-90e7-489a-b52a-0fdd874dac8e)

```suggestion
        class="search-form__input search-form__input--small"
```